scripts/get_pulls_in_range.py

```diff
@@ -131,12 +131,21 @@ def parse_args() -> argparse.Namespace:
     parser.add_argument('-z', '--zephyr-areas', action='store_true',
                         help='''local_path is zephyr; also try to figure
                         out what areas the pull requests affect''')
+    parser.add_argument('-A', metavar='AREA', dest='zephyr_only_areas',
```

Comment:
Long option is missing, `parser.add_argument('-A', '--area', metavar='AREA', dest='zephyr_only_areas',` ?

Reply:
I deliberately did not include a long option, but sure, I can add one.